outputs.tf

```
@@ -1,14 +1,219 @@
+  description = "The private DNS zone for this assessment."
+}
+
+output "assessor_portal_instance_profile" {
+  value       = aws_iam_instance_profile.assessorportal
+  description = "The instance profile for the Assessor Portal instances."
+}
+
+output "assessor_portal_instances" {
+  value       = aws_instance.assessorportal
+  description = "The Assessor Portal instances."
+}
+
+output "assessorportal_security_group" {
+  value       = aws_security_group.assessorportal
+  description = "The security group for the Assessor Portal instances."
+}
+
+output "aws_region" {
+  value       = var.aws_region
+  description = "The AWS region where this assessment environment lives."
+}
+
+output "certificate_bucket_name" {
+  value       = var.cert_bucket_name
+  description = "The name of the S3 bucket where certificate information is stored for this assessment."
+}
+
+output "cloudwatch_and_ssm_agent_security_group" {
+  value       = aws_security_group.cloudwatch_and_ssm_agent
+  description = "A security group for _all_ instances.  Allows access to the VPC endpoint resources necessary for the AWS CloudWatch agent and the AWS SSM agent."
+}
+
+output "debian_desktop_instance_profile" {
+  value       = aws_iam_instance_profile.debiandesktop
+  description = "The instance profile for the Debian desktop instances."
+}
+
+output "debian_desktop_instances" {
+  value       = aws_instance.debiandesktop
+  description = "The Debian desktop instances."
+}
+
+output "debian_desktop_security_group" {
+  value       = aws_security_group.debiandesktop
+  description = "The security group for the Debian desktop instances."
+}
+
+output "efs_client_security_group" {
+  value       = aws_security_group.efs_client
+  description = "A security group that should be applied to all instances that will mount the EFS file share."
+}
+
+output "efs_mount_targets" {
+  value       = aws_efs_mount_target.target
+  description = "The mount targets for the EFS file share."
+}
+
+output "email_sending_domain_certreadrole" {
+  value       = module.email_sending_domain_certreadrole
+  description = "The IAM role that allows for reading the certificate for the email-sending domain."
+}
+
+output "gophish_instance_profile" {
+  value       = aws_iam_instance_profile.gophish
+  description = "The instance profile for the Gophish instances."
+}
+
+output "gophish_instances" {
+  value       = aws_instance.gophish
+  description = "The Gophish instances."
+}
+
+output "gophish_security_group" {
+  value       = aws_security_group.gophish
+  description = "The security group for the Gophish instances."
+}
+
+output "guacamole_accessible_security_group" {
+  value       = aws_security_group.guacamole_accessible
+  description = "A security group that should be applied to all instances that are to be accessible from Guacamole."
+}
+
 output "guacamole_server" {
   value       = aws_instance.guacamole
   description = "The AWS EC2 instance hosting guacamole."
```

Should this be a proper noun?

```suggestion
  description = "The AWS EC2 instance hosting Guacamole."
```

---

Please see commit a22bb6456830b2526ec2ff47b45aa94ba1435e80.